README.md

```diff
@@ -44,7 +44,7 @@ and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/vers
 
 5. If that succeeds, it will end with printing the following command:
    ```
-   source /.../python-zulip-api/.../activate
+   source ./zulip-api-py3-venv/Scripts/activate
```

Comment:
`Scripts` is not available on Linux or OSX. This change does not make sense. It is `bin` over there. So `source /.../python-zulip-api/.../activate` makes more sense.